src/test/java/org/biouno/unochoice/UiAcceptanceTest.java

```diff
@@ -0,0 +1,200 @@
+package org.biouno.unochoice;
+
+import hudson.model.FreeStyleProject;
+import io.github.bonigarcia.wdm.WebDriverManager;
+import org.junit.After;
+import org.junit.Before;
+import org.junit.BeforeClass;
+import org.junit.Rule;
+import org.junit.Test;
+import org.jvnet.hudson.test.JenkinsRule;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.openqa.selenium.chrome.ChromeOptions;
+import org.openqa.selenium.support.ui.Select;
+
+import javax.xml.transform.Source;
+import javax.xml.transform.stream.StreamSource;
+import java.util.List;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertTrue;
+
+public class UiAcceptanceTest {
+    @Rule
+    public JenkinsRule j = new JenkinsRule();
+
+    private WebDriver driver;
+
+    @BeforeClass
+    public static void setUpClass() {
+        WebDriverManager.chromedriver().browserVersion("112").setup();
```

Comment:
I had to put `.browserVersion("112")` here for this to work on Jenkins infrastructure.
On my laptop, I can remove it and it picks up `114` automatically.

This is not ideal, but it is a helpful validation. If it feels hacky, we can close this PR.